src/App.jsx

```diff
@@ -14,33 +24,44 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+function sortGoods(goods, { sortBy: criteria, reverse }) {
+  let sortedGoods;
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  switch (true) {
+    case criteria === SORT_BY_NAME:
```

Comment:
```suggestion
  switch (criteria) {
    case SORT_BY_NAME:
```
Wrong usage of switch block (it doesn't make sense like this, you still do compare in every case like in common if block)